.ci/non-working.txt

```diff
@@ -0,0 +1,3 @@
+bottomhalf
```

Comment:
The suffix `.txt` is redundant. In UNIX-like systems, the suffix does not help a lot.